geojson/types.py

```diff
@@ -0,0 +1,37 @@
+from decimal import Decimal
+from numbers import Real
+from typing import (
+    Callable,
+    List,
+    MutableMapping,
+    Optional,
+    Tuple,
+    TypeAlias,
+    TypeVar,
+    Union,
+)
+
+from geojson.base import GeoJSON
+
+# Python 3.7 doesn't support using isinstance() with Protocols
+# Use checks for (Real, Decimal) instead
+SupportsRound: TypeAlias = Union[Real, Decimal]
```

Comment:
Can we review #188 to see if type checking against Numbers is a better idea than Real or Decimal? This is more in alignment with PEP 3141 and the discussion in #173

Reply:
Hello, thank you for your comment. It's been a while since I suggested this PR, but if I remember correctly, I considered the possibility of using `Number` instead of the `Real`-`Decimal` combination. Please correct me if I am wrong, but as I understand, some members of the Number subset do not support the `SupportsRound` protocol (for example, Complex). So, it is impossible to use them in this case.
The best solution is to use the `SupportsRound` protocol from the `typing` library, but it is only possible in the case of dropping support for Python 3.7.

Reply:
Reopening this as a reminder to implement `typing.SupportsRound` here as part of dropping Python 3.7 support.

Reply:
Also, to be clear, @am1ter, please use `Number` as per https://github.com/jazzband/geojson/pull/188 as @trcmohitmandokhot suggested. Let's not worry about the edge case of complex numbers to whatever extent that might still be a factor.